### PR TITLE
fix(lint): make changes using golangci-lint report

### PR DESCRIPTION
**Pull request by shotonoff**
## What was done?

Fixed lint related issues using golangci-lint report:
* update `.golangci.yml` with upstream version
* format of imports section
* remove dead code
* fix typo/misspelling in comments
* format of indents

## How Has This Been Tested?

Run command `golangci-lint run -c .golangci.yml`

## Breaking Changes

none

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [ ] I have assigned this pull request to a milestone

### CODE REVIEW

File: internal/p2p/conn/secret_connection_test.go [hunk 1]
```diff
@@ -21,6 +20,7 @@ import (
 	"github.com/stretchr/testify/require"
```
Comment (lklimek) [inline internal/p2p/conn/secret_connection_test.go:21]: do we remove empty lines like this?

File: internal/p2p/switch_test.go [hunk 2]
```diff
@@ -15,13 +15,12 @@ import (
 	"testing"
 	"time"
 
-	"github.com/tendermint/tendermint/crypto"
-
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
```
Comment (lklimek) [inline internal/p2p/switch_test.go:21]: do we remove empty lines like this?

**Review by lklimek — CHANGES_REQUESTED** [review]

File: internal/consensus/msgs_test.go [hunk 3]
```diff
@@ -8,12 +8,11 @@ import (
 	"testing"
 	"time"
 
-	"github.com/tendermint/tendermint/crypto"
-
 	"github.com/gogo/protobuf/proto"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
```
[the remark this answers is not in the document]
Comment (shotonoff) [reply internal/consensus/msgs_test.go:14]: I used a blank line to separate import sections between builtin, external and internal deps

File: internal/p2p/switch_test.go (hunk 2 above)
Comment (shotonoff) [reply internal/p2p/switch_test.go:21]: kept empty line for separating

File: internal/p2p/conn/secret_connection_test.go (hunk 1 above)
Comment (shotonoff) [reply internal/p2p/conn/secret_connection_test.go:21]: kept empty line for separating

**Review by shotonoff — COMMENTED** [review]
the content of .golangci.yml took from `master` upstream. I think it would be better to keep the same configuration, otherwise we have to add disabling this checking by ourselves, that will give us extra work

**Review by lklimek — APPROVED** [review]
LGTM

**Pull request merged**